examples/book_translation/translate_latex_book.ipynb

```diff
@@ -224,14 +224,14 @@
     "\n",
     "{sample_translation[1]}\n",
     "'''\n",
-    "    response = openai.Completion.create(\n",
+    "    response = openai.chat.completions.create(\n",
     "        prompt=prompt,\n",
-    "        engine=engine,\n",
+    "        model=engine,\n",
     "        temperature=0,\n",
     "        top_p=1,\n",
     "        max_tokens=1500,\n",
     "    )\n",
-    "    result = response['choices'][0]['text'].strip()\n",
+    "    result = response.choices[0].text.strip()\n",
```

Comment:
response['choices'][0]['message']['content']

Reply:
changed to response.choices[0].message.content.strip()